src/main/java/WebRTCConf.java

```diff
@@ -0,0 +1,244 @@
+           int roomId = getRoomID();
+            
+        	//Getting User's Selection
+        	Scanner sc = new Scanner(System.in);
+        	System.out.println("Please Select a Browser Combination to initiate the WebRTC test on BrowserStack : ");
+        	System.out.println("1 : Press 1 to initiate WebRTC test on Google Chrome and Firefox Browser Combination ");
+        	System.out.println("2 : Press 2 to initiate WebRTC test on Edge and Safari Browser Combination ");
+        	System.out.println("3 : Press 3 to initiate WebRTC test on Android and iOS Combination");
+        	int userSelection  = sc.nextInt();
+        	sc.close();
+        	Thread t1 = null;
+        	Thread t2 = null;
+        	
+        	if(userSelection == 1) {
+        		System.out.println("Your test would execute on Chrome-Firefox Browser Combination");
+        		//Creating capabilities for Chrome Browser.
+        		ChromeOptions chromeConfiguration = getChromeConfiguration();
+        		//Creating capabilities for FireFox Browser.
+        		ChromeOptions fireFoxConfiguration = getFireFoxConfiguration();
+        		
+        		//Creating the WebRTC Room on Chrome Browser.
+        		System.out.println("Creating a Room on Chrome Browser");
+                t1 = new Thread(new WebRTCTestRunner(chromeConfiguration, String.valueOf(roomId),false,20000, userSelection));
+                t1.start();
+                //Waiting for other user to join the above created WebRTC room.
+                Thread.sleep(15000);
+                
+                // Joining the above created Room on FireFox Browser.
+                System.out.println("Joining the Room on Firefox Browser");
+                t2 = new Thread(new WebRTCTestRunner(fireFoxConfiguration,String.valueOf(roomId),true,20000, userSelection));
+                t2.start();
+        		
+        	}else if(userSelection == 2) {
+        		System.out.println("Your test would execute on Edge-Safari Browser Combination");
+        		//Creating capabilities for Edge Browser.
+                ChromeOptions edgeConfiguration = getEdgeConfiguration();
+                //Creating capabilities for Safari Browser.
+                ChromeOptions safariConfiguration = getSafariConfiguration();
+
+                //Creating the WebRTC Room on Edge Browser.
+                System.out.println("Creating a Room on Edge Browser");
+                t1 = new Thread(new WebRTCTestRunner(edgeConfiguration, String.valueOf(roomId),false,20000, userSelection));
+                t1.start();
+                
+                //Waiting for other user to join the above created WebRTC room.
+                Thread.sleep(15000);
+                
+                // Joining the above created Room on Safari Browser.
+                System.out.println("Joining the Room on Safari Browser");
+                t2 = new Thread(new WebRTCTestRunner(safariConfiguration,String.valueOf(roomId),true,20000, userSelection));
+                t2.start();
```

Comment:
can we have all this code as a common separate function? and call it with different parameter. this way we can reduce the repeated code

Reply:
done